<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticeWindow</class>
 <widget class="QDialog" name="PracticeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backArrow">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font:85px;
	color:#8E8E8E;
	font-weight:2px;
	qproperty-alignment: AlignCenter;
	font-family: fontSpaceGrotesk;
	background-color:transparent;
	border:none;
}
QPushButton:hover{
	color: #8BB5FF;
}</string>
   </property>
   <property name="text">
    <string>‚Üê</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleTextBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>900</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-alignment: AlignCenter;
font-weight: bold;
font-family: fontSpaceGrotesk;
color:black;
font:60px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="knowTextBox">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>391</y>
     <width>124</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:#8BCD9A;
	font:48px;
	qproperty-alignment: AlignCenter;
	font-family: fontSpaceGrotesk;
	border: none;
	background-color: transparent;
}

QPushButton:hover{
	color:#2bb54a;
}</string>
   </property>
   <property name="text">
    <string>Know</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dontKnowTextBox">
   <property name="geometry">
    <rect>
     <x>1091</x>
     <y>391</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:#955F91;
	font:48px;
	qproperty-alignment: AlignCenter;
	font-family: fontSpaceGrotesk;
	border: none;
	background-color: transparent;
}

QPushButton:hover{
	color:#dd17d0;
}</string>
   </property>
   <property name="text">
    <string>Don't Know</string>
   </property>
  </widget>
  <widget class="QLabel" name="flashCardTextBox">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>345</y>
     <width>330</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-wordWrap: true;
qproperty-alignment: AlignCenter;
color:white;
font:45px;
font-weight:bold;
font-family: fontSpaceGrotesk;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="finishTextBox">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>580</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	qproperty-alignment: AlignCenter;
	color: black;
	font:25px;
	font-weight:bold;
	font-family: fontSpaceGrotesk;
	background-color: transparent;
	border: none;
}
QPushButton:hover{
	color: #8E8E8E;
}</string>
   </property>
   <property name="text">
    <string>Finish Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="flashCard">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>302</y>
     <width>418</width>
     <height>224</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="meaning">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>255</y>
     <width>330</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-wordWrap: true;
qproperty-alignment: AlignCenter;
color:black;
font:50px;
font-family: fontSpaceGrotesk;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>It means:</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>flashCard</zorder>
  <zorder>backArrow</zorder>
  <zorder>titleTextBox</zorder>
  <zorder>knowTextBox</zorder>
  <zorder>dontKnowTextBox</zorder>
  <zorder>finishTextBox</zorder>
  <zorder>flashCardTextBox</zorder>
  <zorder>meaning</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
